sales.html

```diff
@@ -0,0 +1,9 @@
+<html>
+  <head>
+  </head>
+  <body>
+    <link href='style.css' ref='stylesheet' type='text/css'>
+    <script src='app.js'></script>
+    <link href='style.css' ref='stylesheet' type='text/css'>
```

Comment:
you've imported style.css twice, you only need to do it once. And it's best-practice to import the styles in the head tags so they load first.